alphadia/outputtransform/search_plan_output.py

```diff
@@ -593,7 +594,15 @@ class LFQOutputConfig:
                 group_column=quantlevel_config.quant_level,
             )
 
-            lfq_results[quantlevel_config.level_name] = lfq_df
+            if quantlevel_config.level_name != "pg":
+                annotate_df = psm_df.groupby(
+                    quantlevel_config.quant_level, as_index=False
+                ).agg({"pg": "first", "sequence": "first"})
```

Comment:
why do we aggregate on the sequence here?
We usually use the `mod_seq_charge` hash or the `mod_seq_hash`. Sequence level is rather uncommon. Should we make it more explicit that `mod_seq_charge` is aggregated on `sequence`, `mods `and `charge` and `mod_seq_hash` only on `sequence`, `mods `